main.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func main() {
-	cmd.InitCommandFlags()
+	cmd.InitCommands()
```

Comment:
Instead of this I would like a `cmd.RootCmd()` that does the init. That way we don't forget to call it :)